doc/user guide.md

```diff
@@ -0,0 +1,346 @@
+  * **"Prometheus"**: Are you using prometheus to track tendermint metrics?
+  * **"PrometheusListenAddr"**: If so, on what port?
+  * **"MaxOpenConnections"**: What is the maximum number of simultaneous connections you'd like to allow on prometheus?
+  * **"Namespace"**: What namespace would you like to use for prometheus?
+
+### Open Files Calculation
+Pocket Core operation requires an elevated Ulimit:
+```
+({ulimit -Sn} >= {MaxNumInboundPeers} + {MaxNumOutboundPeers} + {GRPCMaxOpenConnections} + {MaxOpenConnections} + {Desired Concurrent Pocket RPC connections} + {100 (Constant number of wal, db and other open files)}
+```
+### Genesis File
+Located:
+`$HOME/.pocket/config/genesis.json`
+
+[Testnet Genesis File](https://raw.githubusercontent.com/pokt-network/pocket-network-genesis/master/testnet/genesis.json)
+
+[Mainnet Genesis File](https://raw.githubusercontent.com/pokt-network/pocket-network-genesis/master/mainnet/genesis.json)
+
+Use pocket core flags --mainnet or --testnet to automatically write
+### Chains.json
+Use the CLI or Manually Edit: `$HOME/.pocket/config/chains.json`
+
+Relay Chain ID's can be found [here](https://forum.pokt.network/t/supportedblockchains/607)
+```
+[
+  {
+    "id": "0002",
+    "url": "http://eth-geth.com",
+    "basic_auth": {
+      "username": "",
+      "password": ""
+    }
+  }
+]
+```
+## Operation
+Operating a Validator requires (at a minimum) some prerequisite basic knowledge of the Pocket Network. 
+
+*This section will cover the basics of:*
+- Slashing and Jailing
+- Force Unstake
+- Economic Incentives
+
+**Slashing And Jailing**
+
+Jailing and Slashing are high level protocol concepts:
+
+*Jailing* a Validator removes them from both protocol service and consensus. 
+
+*Slashing* a Validator burns a percentage of the 'Staked Tokens'
+
+A Validator is jailed and subsequently slashed for not signing (or incorrectly signing) block proposals. More often than not, this is the reason why Validators are jailed. 
+
+Common reasons for not signing blocks are addressed [here](https://github.com/pokt-network/pocket-core/issues/1092)
+
+If a Validator is jailed for too long it will be forcibly removed by the protocol and all Staked Tokens burned
+
+NOTE: `pocket query params` to see protocol level values like `max_jailed_blocks`
+
+**Force Unstake**
+
+If a Validator falls below the minimum stake (due to slashing) it will be forcibly removed by the protocol and all Staked Tokens burned. This feature of the protcol highlights the importance of staking 'well above' the minimum stake.
+
+**Economic Incentives**
+
+For providing infrastructure access to applications, Validators are rewarded proportional to the work they provide. Pocket Core attempts to send a *Claim* and subsequent *Proof* transaction automatically after the `proof_waiting_period` elapses. If both transactions are successful, Tokens are minted to the address of the Validator.
+
+## Resources
+[Pocket Core CLI Spec](cli-interface-spec.md)
```

Comment:
This is now contained and split into a cli-interface-spec directory rather than a single file see #1232

Reply:
Well I guess it depends which PR is merged first haha JK, I'll fix it